src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -71,7 +71,7 @@ type Props = {
     tooltips: { [componentIdentifier: tooltipComponentIdentifierType]: React.Node },
     trackingProps: sharedLinkTrackingType,
     triggerCopyOnLoad?: boolean,
-};
+} & InjectIntlProvidedProps;
```

Comment:
You already have `intl: IntlShape`, so you don't need `InjectIntlProvidedProps`

Reply:
when would I want to use one over the other?

Reply:
I think IntlShape is generally cleaner

Reply:
i like that!